runtime/py.yml

```diff
@@ -22,4 +23,4 @@ dependencies:
   - xgboost=1.2.0=py38h950e882_0
   - cloudpickle=1.3.0
   - tsfresh=0.17.0
-  - pytorch-lightning=1.1.2
```

Comment:
@Fissium Can you add a new line here for `tensorflow-addons` instead of replacing the line for `pytorch-lightning`?